src/runtime/node/process/_process.ts

```diff
@@ -226,3 +226,12 @@ process.chdir = function (dir) {
 process.umask = function () {
   return 0;
 };
+
+// https://nodejs.org/api/process.html#processhrtime
+process.hrtime = function () {
+  return [0, 0];
+};
```

Comment:
hey @alan-agius4, I think you actually want something closer to this:

```suggestion
process.hrtime = function (startTime) {
  const now = Date.now();
  // millis to seconds
  const seconds = Math.trunc(now / 1_000);
  // convert millis to nanos
  const nanos = (now % 1_000) * 1_000_000;
  
  if (startTime) {
    let diffSeconds = seconds - startTime[0];
    let diffNanos = nanos - startTime[0];
    
    if (diffNanos < 0) {
      diffSeconds = diffSeconds - 1;
      diffNanos = 1_000_000_000 + diffNanos;
    }
    return [diffSeconds, diffNanos]
  }
  
  return [seconds, nanos];
};
```

Reply:
Thanks @IgorMinar 😀